ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c

```diff
@@ -6460,6 +6463,9 @@ int bdb_public_cursor_op(dbi_cursor_t *cursor,  dbi_op_t op, dbi_val_t *key, dbi
     DBT bdb_data = {0};
     int rc = 0;
 
+    if (bdb_cur == NULL)
```

Comment:
Braces around the return condition IE if (..) { return .... };

Reply:
ok. This change is now committed.